scopesim/effects/surface_list.py

```diff
@@ -1,125 +1,33 @@
-
 import numpy as np
-
 from astropy import units as u
+import matplotlib.pyplot as plt
 
-from synphot import SourceSpectrum, SpectralElement
-from synphot.models import Empirical1D
-
-from .. import rc
-from .. import utils
-from ..utils import quantify
-from ..base_classes import SourceBase, ImagePlaneBase, FieldOfViewBase
-from ..optics.radiometry import RadiometryTable
-from .effects import Effect
 from .ter_curves import TERCurve
+from ..optics import radiometry_utils as rad_utils
+from ..optics.surface import PoorMansSurface, SpectralSurface
+from ..utils import quantify, from_currsys
 
 
-class SurfaceList(Effect):
-    """
-    A Effect object containing a list of all surfaces in an optical element
-
-    This calls is essentially a wrapper for the RadiometryTable class, which
-    contains all the functionality for creating and combining multiple optical
-    surfaces
-
-
-    Parameters
-    ----------
-    filename : str
-        Path to file containing
-    table : astropy.Table
-
-    array_dict : dict
-
-
-    Input format
-    ------------
-
-    """
+class SurfaceList(TERCurve):
```

Comment:
Yep. To be honest, it should have been the logical conclusion in the beginning anyway, as the surface list should have the same effect on the Source spectra as any individual surface. 

The reason it wasn't like that was because I convinced myself that I needed a "system SurfaceList" that collected and combined all surfaces and surface_lists in the whole optical train.